2.4.0/Dockerfile

```diff
@@ -28,8 +28,8 @@ RUN set -eux; \
 # Verify that gosu binary works
     gosu nobody true
 
-ARG GPG_KEY=5167DE337E7370373499FC1DA4A672F11B5050C8
-ARG DISTRO_NAME=apache-storm-2.3.0
+ARG GPG_KEY=51379DA8A7AE5B02674EF15C134716AF768D9B6E
```

Comment:
The key has changed again, but it is one of the official signing keys: https://downloads.apache.org/storm/KEYS.